src/ldp/operations/GetOperationHandler.ts

```diff
@@ -25,6 +26,13 @@ export class GetOperationHandler extends OperationHandler {
 
   public async handle(input: Operation): Promise<ResponseDescription> {
     const body = await this.store.getRepresentation(input.target, input.preferences);
+    if (this.isAuthorized(input)) {
+      input.authorization.addMetadata(body.metadata);
+    }
     return new OkResponseDescription(body.metadata, body.data);
   }
+
+  private isAuthorized(input: Operation): input is AuthorizedOperation {
+    return typeof (input as AuthorizedOperation).authorization === 'object';
+  }
```

Comment:
This code is repeated?
Either put in a utility (inside `AuthorizedOperation.ts` would be good) or in base class.